templates/http-py-simple/content/spin.toml

```diff
@@ -0,0 +1,17 @@
+spin_manifest_version = 2
+
+[application]
+authors = ["{{authors}}"]
+description = "{{project-description}}"
+name = "{{project-name}}"
+version = "0.1.0"
+
+[[trigger.http]]
+route = "{{http-path}}"
+component = "{{project-name | kebab_case}}"
+
+[component.{{project-name | kebab_case}}]
+source = "app.wasm"
+[component.{{project-name | kebab_case}}.build]
+command = "componentize-py -w spin-http componentize -p . -p $(python3 -m site --user-site) -p $(python3 -c 'import site; print(site.getsitepackages()[0])') app -o app.wasm"
```

Comment:
I'll try this myself when I get a chance, but have you tried this when using a virtual environment like the main README.md in this repo describes?  Just want to make sure it works in both the virtual environment and host environment scenario.

Reply:
I did not try the virtual environment but will make sure I test it later today/tomorrow

Reply:
I tested and it works from local but I think I need to remove the `-p $(python3 -m site --user-site)` as that still points outside the virtual env.

Reply:
Also, should we just have a requirements.txt file tying down the spin_sdk and componentize-py versions?

Reply:
Yeah, that's a good idea.